metricbeat/module/haproxy/stat/stat.go

```diff
@@ -65,13 +67,13 @@ func (m *MetricSet) Fetch() ([]common.MapStr, error) {
 
 	hapc, err := haproxy.NewHaproxyClient(m.statsAddr)
 	if err != nil {
-		return nil, fmt.Errorf(fmt.Sprintf("HAProxy Client error: %s", err))
+		return nil, fmt.Errorf("HAProxy Client error: %s", err)
```

Comment:
errors.New would also work here. But that is a minor detail which can be tackled later.